panel/src/libraries/dayjs-pattern.test.js

```diff
@@ -69,26 +69,30 @@ describe("dayjs.pattern.format()", () => {
 describe("dayjs.pattern.interpret()", () => {
   const data = {
     "YYYY-MM-DD": {
+      pattern: "YYYY-MM-DD",
       inputs: ["2020-08-01", "20-8-1", "20-8", "20/8"],
       result: "2020-08-01"
     },
     "MM/DD/YY HH:mm": {
+      pattern: "MM/DD/YY HH:mm",
       inputs: ["08/02/15 08:03", "8/2/15 8:3", "8.2.15 8.3"],
       result: "08/02/15 08:03"
     },
     "YY.M.D": {
+      pattern: "YY.M.D",
       inputs: ["20.1.1", "20.1", "20"],
       result: "20.1.1"
     },
     "HH:mm:ss": {
+      pattern: "HH:mm:ss",
       inputs: ["19:03:22", "19.3.22"],
       result: "19:03:22"
     }
   };
 
   for (const test in data) {
     it(test, () => {
-      const pattern = dayjs.pattern(test);
+      const pattern = dayjs.pattern(data[test].pattern);
```

Comment:
@bastianallgeier what's the purpose here not to use the key as pattern but add the same as sub-key?

Reply:
I wanted to add another test for YYYY-MM-DD but with a different result. This is not possible so far. 

I didn't add the test, because the interpretation isn't able to handle it, but I still wanted to prepare it for more such tests. 

The case here is: 

```js
    "YYYY-MM-DD with just a year": {
      pattern: "YYYY-MM-DD",
      inputs: ["2012"],
      result: "2012-01-01"
    },
```

Reply:
ok :)